backend/src/AllStars.Application/Services/Dutch/DutchService.cs

```diff
@@ -7,92 +7,64 @@ namespace AllStars.Application.Services.Dutch;
 {
-    private readonly IDutchRepository _dutchRepository = dutchRepository;
-    private readonly IUserRepository _userRepository = userRepository;
-    private readonly ILogRepository _logRepository = logRepository;
-
-    public async Task<IEnumerable<DutchScore>> GetUserScores(string name, CancellationToken token)
-    {
-        var scores = await _dutchRepository.GetUserScores(name, token);
-        return scores;
-    }
+    public async Task<IEnumerable<DutchScore>> GetUserScores(string name, CancellationToken token) 
+        => await dutchRepository.GetUserScores(name, token);
 
     public async Task<IEnumerable<DutchScore>> GetAll(CancellationToken token)
-    {
-        var scores = await _dutchRepository.GetAll(token);
-        return scores;
-    }
+        => await dutchRepository.GetAll(token);
+
 
     public async Task<bool> UpdateOne(Guid gameId, string nickName, int points, CancellationToken token)
     {
-        var score = await _dutchRepository.GetOneScore(gameId, nickName, token);
+        var score = await dutchRepository.GetOneScore(gameId, nickName, token);
         if (score is null)
         {
             return false;
         }
 
-        await _dutchRepository.UpdateOne(score, points, token);
-
+        await dutchRepository.UpdateOne(score, points, token);
 
         return true;
     }
 
     public async Task CreateMany(CreateDutchGameCommand createDutchGameCommand, CancellationToken token)
     {
         var nickNames = createDutchGameCommand.ScorePairs
-            .Select(x => x.NickName);
+            .Select(x => x.NickName)
+            .ToList();
 
-        var users = await _userRepository.GetManyAsync(nickNames, token);
+        var users = await userRepository.GetManyAsync(nickNames, token);
 
-        if (users.Count() != nickNames.Count())
+        if (users.Count != nickNames.Count)
         {
-            var foundUsers = users.Select(u => u.Nickname);
-            var missingNickNames = nickNames.Except(foundUsers);
-            var missingNickNamesMessage = string.Join(", ", missingNickNames);
+            var foundUserNames = users.Select(u => u.Nickname);
+            var missingNickNames = nickNames.Except(foundUserNames);
 
-            throw new InvalidOperationException($"Cannot create game. Some users where not found: {missingNickNamesMessage}");
+            throw new InvalidOperationException($"Cannot create game. Some users where not found: {string.Join(", ", missingNickNames)}");
         }
 
-        var game = new DutchGame()
+        var game = new DutchGame
         {
             Id = Guid.NewGuid(),
             Date = DateTime.Now,
             Comment = createDutchGameCommand.Comment
         };
-
-        var sortedScorePairs = createDutchGameCommand.ScorePairs
-            .OrderBy(c => c.Score)
-            .ToList();
-
-        var scores = new List<DutchScore>();
-        int currentRank = 1;
-        int previousScore = int.MinValue;
```

Comment:
are you sure your code is working when two players have the same amount of points?

For A : 15, B :14, C:15 We want A: Positioon = 2 B Position = 1 and C : Position = 2. I think your code will give randomly 2 and 3 across a and c